daprdocs/content/en/developing-applications/building-blocks/state-management/query-state-store/query-sqlserver-store.md

```diff
@@ -24,42 +34,43 @@ SELECT * FROM states WHERE [Key] LIKE 'myapp||%'
 
 The above query returns all rows with id containing "myapp||", which is the prefix of the state keys.
 
-## 3. Get specific state data
+## Get specific state data
 
 To get the state data by a key "balance" for the application "myapp", use the query:
 
 ```sql
 SELECT * FROM states WHERE [Key] = 'myapp||balance'
 ```
 
-Then, read the **Data** field of the returned row.
-
-To get the state version/ETag, use the command:
+Read the **Data** field of the returned row. To get the state version/ETag, use the command:
 
 ```sql
 SELECT [RowVersion] FROM states WHERE [Key] = 'myapp||balance'
 ```
 
-## 4. Get filtered state data
+## Get filtered state data
 
 To get all state data where the value "color" in json data equals to "blue", use the query:
 
 ```sql
 SELECT * FROM states WHERE JSON_VALUE([Data], '$.color') = 'blue'
 ```
 
-## 5. Read actor state
+## Read actor state
 
 To get all the state keys associated with an actor with the instance ID "leroy" of actor type "cat" belonging to the application with ID "mypets", use the command:
 
 ```sql
 SELECT * FROM states WHERE [Key] LIKE 'mypets||cat||leroy||%'
 ```
 
-And to get a specific actor state such as "food", use the command:
+To get a specific actor state such as "food", use the command:
 
 ```sql
 SELECT * FROM states WHERE [Key] = 'mypets||cat||leroy||food'
 ```
 
-> **WARNING:** You should not manually update or delete states in the store. All writes and delete operations should be done via the Dapr runtime.
+{{% alert title="Warning" color="warning" %}}
+You should not manually update or delete states in the store. All writes and delete operations should be done via the Dapr runtime.
```

Comment:
```suggestion
You should not manually update or delete states in the store. All writes and delete operations should be done via the Dapr runtime. The one exception to this, is that it is often required to delete actor records in a state store, _once you know that these are no longer in use_, to prevent a build up of unused actor instance that may never be loaded again.
```